### DISCUSSION TITLE
Getting comments instead of program for some simple requests

### DISCUSSION THREAD

**Discussion by tarunvignesh1** [community/community#7424] [Copilot Conversations]
I tried to check with some random requests like
Language used :C++

> //return a longest palindromic possible string from a larger string
> // e.g. "abccba" -> "abccba"
> // "abccbac" -> "abccba"
> // "abccbacb" -> "abccbacb"
> // "abccbacbac" -> "abccbacbac"
> // "abccbacbacb" -> "abccbacbacb"
> // "abccbacbacbac" -> "abccbacbacbac"
> // "abccbacbacbacb" -> "abccbacbacbacb"
> // "abccbacbacbacbac" -> "abccbacbacbacbac"
> // "abccbacbacbacbacb" -> "abccbacbacbacbacb"
> // "abccbacbacbacbacbac" -> "abccbacbacbacbacbac"
> // "abccbacbacbacbacbacb" -> "abccbacbacbacbacbacb"
> // "abccbacbacbacbacbacbac" -> "abccbacbacbacbacbacbac"
> // "abccbacbacbacbacbacbacb" -> "abccbacbacbacbacbacbacb"
> // "abccbacbacbacbacbacbacbac" -> "abccbacbacbacbacbacbacbac"

**it returns a set of comments of example I/O pairs**

**Comment by trick4u** [comment]
same here..could not figure out the reason
